modules/events/events.slice.ts

```diff
@@ -100,6 +110,13 @@ export const selectFilteredSortedEvents = createSelector(
     (events, eventFilters, sortingFilters) => searchAndSortWithEmbeddedField(events, sanitizeFilters(eventFilters), sortingFilters.ascending, 'meta', sortingFilters.sortKey)
 )
 
+export const selectMultipleFilteredSortedEvts = createSelector(
+    selectEventsByDateRange,
+    (state: AppState) => state.eventsMultipleFilters.events,
+    (state: AppState) => state.eventSorting,
+    (events, eventMultipleFilters, sortingFilters) => filterByEnumAndSort(events, eventMultipleFilters, 'event', sortingFilters.ascending, 'meta', sortingFilters.sortKey)
+)
+
 export const selectEventTotals = createSelector(
     selectFilteredEvents,
```

Comment:
This should be 'selectMultipleFilteredSortedEvts' if I am not mistaken. Unless 'selectFilteredEvents' uses the 'eventsMultipleFilters' filters. Just make sure the totals match the filter

Reply:
Okay